cookbook/acceptance-testing-with-selenium/README.md

```diff
@@ -117,28 +105,39 @@ It is a good practice to always wait:
-	compare the text value of `TextareaInfoLabel` asynchronously. The assertion passes if the 
-	text is found within 5 seconds, otherwise it fails.
+    - An example can be found in the method `TextareaPage_WriteToTextArea` in KitchenSink 
+    Tests (see [TextareaPageTest.cs lines 28-38](https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionString/TextareaPageTest.cs#L28-L38)). The method `WaitForText()` is used to 
+    compare the text value of `TextareaInfoLabel` asynchronously. The assertion passes if the 
+    text is found within 5 seconds, otherwise it fails.
 
 - Wait for a button to be present before you click on that button
 
-	- An example can be found in the method `ButtonPage_RegularButton` in the KitchenSink 
-	tests (see [ButtonPageTest.cs lines 29-46](https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionNumber/ButtonPageTest.cs#L29-L46)). The method `WaitUntil()` is used to 
-	asynchronously check the state of the `Displayed` property of a button. It halts the test 
-	for a default maximum time of 10 seconds. If the button is not displayed within that 
-	time, it throws an exception.
+    - An example can be found in the method `ButtonPage_RegularButton` in the KitchenSink 
+    tests (see [ButtonPageTest.cs lines 29-46](https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionNumber/ButtonPageTest.cs#L29-L46)). The method `WaitUntil()` is used to 
+    asynchronously check the state of the `Displayed` property of a button. It halts the test 
+    for a default maximum time of 10 seconds. If the button is not displayed within that 
+    time, it throws an exception.
 
 - Wait for presence of an input field before typing in it and wait for text to be present in label
 
-	- An example can be found in the method `TextPage_TextPropagationOnUnfocus` in the 
-	KitchenSink tests (see [TextPageTest.cs lines 28-38](https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionString/TextPageTest.cs#L28-L38)). This test mixes the other examples 
-	presented above. The method `WaitUntil()` is used here to asynchronously wait for the 
-	input to be `Displayed`. Notice that the following response check is also done 
-	asynchrously using `WaitForText`.
+    - An example can be found in the method `TextPage_TextPropagationOnUnfocus` in the 
+    KitchenSink tests (see [TextPageTest.cs lines 28-38](https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionString/TextPageTest.cs#L28-L38)). This test mixes the other examples 
+    presented above. The method `WaitUntil()` is used here to asynchronously wait for the 
+    input to be `Displayed`. Notice that the following response check is also done 
+    asynchrously using `WaitForText`.
+
+## Run your first test
+
+Clone KitchenSink repo from the StarcounterApps organisation on GitHub.
```

Comment:
I know you didn't write this, but it would be helpful with a link to the repo here